Listener/RestListener.php

```diff
@@ -63,7 +67,12 @@ public function onKernelRequest(GetResponseEvent $event)
 
             if ($locale !== null) {
                 $request->setLocale($locale);
+            } else {
+                $request->setLocale(
+                    $request->getPreferredLanguage($this->locales)
```

Comment:
Let's check whether this is really set-up. Currently we could get `null` if `locales` are not configured and no header is passed. This could not work correctly if other Listeners are configured before this one to set the request locale

Reply:
Let's also check if any of `$this->locales` is really provided.

This would make this functionality opt-in. Currently, with no configuration, Symfony would choose first language in the header, which could make some difficulties later if not whitelisted